### PR TITLE
Supervisor manage courses

### PR DESCRIPTION
**Pull request by binhthanhnguyen**

### CODE REVIEW

File: app/views/rails_admin/main/show_course.html.erb [hunk 1]
```diff
@@ -0,0 +1,54 @@
+<div>
+  <ul class="nav nav-tabs" role="tablist">
+    <li role="presentation" class="active">
+      <a href="#overview" aria-controls="overview" role="tab" data-toggle="tab">
+        <%= t "admin.tabs.overview" %>
+      </a>
+      </li>
```
Comment (ttuan) [inline app/views/rails_admin/main/show_course.html.erb]: ident

**Comment by vietanhbkaaa** [comment]
OK

File: app/views/rails_admin/main/show_course.html.erb [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        </dd>
+        <dt>
+          <span class="label label-info name_field string_type">
+            <%= t "admin.course.description" %>
+          </span>
+        </dt>
+        <dd class="well">
+          <%= @course.description %>
+        </dd>
+         <dt>
```
Comment (ttuan) [inline app/views/rails_admin/main/show_course.html.erb]: ident

**Comment by vietanhbkaaa** [comment]
OK

**Pull request merged**